openest/test/curves/test_smart_linextrap.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from openest.generate.smart_curve import *
+from openest.curves.smart_linextrap import *
```

Comment:
In the two lines above I'd change wild card import to explicitly import what's needed. This would make it easier to understand test.